IDEAS/Buildings/Components/Window.mo

```diff
@@ -250,6 +267,11 @@ equation
     connect(heaCapGla.port, layFra.port_a) annotation (Line(points={{16,-12},{16,
             -12},{16,80},{10,80}},  color={191,0,0}));
   end if;
+  connect(theBri.port_b, layFra.port_a) annotation (Line(points={{10,40},{16,40},
```

Comment:
Is there a reason why the ports have been switched? I suggest to keep same convention within 1 component.